boss-ws/src/main/java/org/genomebridge/boss/http/objectstore/GCSObjectStore.java

```diff
@@ -111,6 +114,37 @@ private PrivateKey getKey() throws Exception {
         return mKey;
     }
 
+    @Override
+    public URI generateResumableUploadURL(String objectName) {
+        try{
+            URI uri = generateResumableURI(StringUtils.deleteWhitespace(objectName));
```

Comment:
Remove deleteWhitespace -- just pass the objectName.  We're assuming, for now, that object keys don't need URL-encoding.  That means they won't contain white space.  (And deleteWhitespace doesn't handle the general case of characters that are illegal in URLs, anyway.)